### PR TITLE
Fixed due_datetime formatting

### PR DESCRIPTION
**Pull request by X-pech**
Quick fix of #2489

### CODE REVIEW

**Comment by X-pech** [comment]
Also added an option to specify due_string parameter language by 2-character code

File: packages/nodes-base/nodes/Todoist/GenericFunctions.ts [hunk 1]
```diff
@@ -6,11 +6,19 @@ import {
 	IExecuteFunctions,
 	IHookFunctions,
 	ILoadOptionsFunctions,
+	returnJsonArray,
```
Comment (krynble) [inline packages/nodes-base/nodes/Todoist/GenericFunctions.ts:9]: Could you please remove this import? I think it was added by mistake.

File: packages/nodes-base/nodes/Todoist/GenericFunctions.ts [hunk 2]
```diff
@@ -6,11 +6,19 @@ import {
...
 } from 'n8n-core';
 
 import {
 	IDataObject, NodeApiError,
 } from 'n8n-workflow';
+import { Console } from 'console';
```
Comment (krynble) [inline packages/nodes-base/nodes/Todoist/GenericFunctions.ts:15]: Same for this one, could you please remove?

**Review by krynble — CHANGES_REQUESTED** [review]
Hey @X-pech 

Thanks a lot for your contribution!

I requested 2 small changes so we can proceed with merging your PR.

Can you please fix these?

**Review by krynble — APPROVED** [review]

**Comment by janober** [comment]
Thanks a lot. Got merged.

**Comment by janober** [comment]
Got released with `n8n@0.154.0`

**Pull request merged**